<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintPreview</class>
 <widget class="QDialog" name="PrintPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Preview</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="GroupBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="PixmapLabel" name="pixmapLabel" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="scaledContents" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="GroupBox2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Modify</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkInvert">
        <property name="text">
         <string>&amp;Invert Colors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkMirror">
        <property name="text">
         <string>&amp;Mirror</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkFlip">
        <property name="text">
         <string>&amp;Flip</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="PrintButtonGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Print Dimensions</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,1,0,1,1">
      <item>
       <widget class="QCheckBox" name="maximum">
        <property name="text">
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="centered">
        <property name="text">
         <string>Centered</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel1">
        <property name="text">
         <string>Side Margins:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sidemargins"/>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel2">
        <property name="text">
         <string>Top and Bottom
 Margins</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="topbottommargins"/>
      </item>
      <item>
       <widget class="QCheckBox" name="manualfeed">
        <property name="text">
         <string>Manual Feed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="windowTitle">
        <string/>
       </property>
       <property name="text">
        <string>&amp;Print</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Discard</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>PixmapLabel</class>
   <extends>QWidget</extends>
   <header>pixmaplabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkInvert</tabstop>
  <tabstop>checkMirror</tabstop>
  <tabstop>checkFlip</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <includes>
  <include location="global">qimage.h</include>
  <include location="global">qpixmap.h</include>
  <include location="global">paletteview.h</include>
  <include location="global">pixmaplabel.h</include>
  <include location="global">qrect.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>PrintPreview</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PrintPreview</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkInvert</sender>
   <signal>toggled(bool)</signal>
   <receiver>PrintPreview</receiver>
   <slot>invertColors(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkMirror</sender>
   <signal>toggled(bool)</signal>
   <receiver>PrintPreview</receiver>
   <slot>mirror(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFlip</sender>
   <signal>toggled(bool)</signal>
   <receiver>PrintPreview</receiver>
   <slot>flip(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
